app/controllers/v0/sessions_controller.rb

```diff
@@ -35,7 +35,13 @@ def saml_callback
 
     def persist_session_and_user!
       @session = Session.new(user_attributes.slice(:uuid))
-      @current_user = User.find(@session.uuid) || Decorators::MviUserDecorator.new(User.new(user_attributes)).create
+
+      if (user_attributes['level_of_assurance'] == 'loa1')
```

Comment:
This could be a bit more DRY by doing something like
```
@current_user = User.find(@session.uuid) || User.new(user_attributes)
if @current_user.loa1?
  @current_user = Decorators::MviUserDecorator.new(@current_user).create
end
```